glib/Idle.cs

```diff
@@ -32,7 +32,7 @@ namespace GLib {
 	public class Idle {
 
 		[UnmanagedFunctionPointer (CallingConvention.Cdecl)]
-		delegate bool IdleHandlerInternal ();
+		delegate bool IdleHandlerInternal (IntPtr ptr);
```

Comment:
I'm pretty sure native counterpart is taking pointer, hence IntPtr...
In theory this should be used to store pointer to data object, but int is enough for our use case...